Source/Models/BrickSectionDataSource.swift

```diff
@@ -49,7 +49,7 @@ extension BrickSection {
         return brickSection.numberOfBricks(in: collection)
     }
 
-    internal func brickAndIndex(at indexPath: IndexPath, in collection: CollectionInfo) -> (Brick, Int)? {
+    internal func brickAndIndex(atIndexPath indexPath: IndexPath, in collection: CollectionInfo) -> (Brick, Int)? {
```

Comment:
there is also a `func brickAndIndex(atIndex brickIndex: Int, in collection: CollectionInfo) -> (Brick, Int)?`, so the `atIndexPath` is to make the difference